index.js

```diff
@@ -12,7 +12,7 @@ const getCerts = require(path.resolve(__dirname, "certs.js")).getCerts
 /* CONFIGURE THE SERVER */
 
 // SSL certificate
-const createServer = () => {
+const createServer = (customDomain = "") => {
```

Comment:
(\*undefined)
```suggestion
const createServer = (customDomain = undefined) => {
```
OR
```suggestion
const createServer = (customDomain) => {
```
OR
```suggestion
const createServer = (domain = "localhost") => {
```
In this latter case change also line 25 from `customDomain` to `domain`.